### DISCUSSION TITLE
Access denied for query to access productVariant.quantityAvailable field

### DISCUSSION THREAD

**Discussion by marcgug** [Shopify/storefront-api-feedback#223] [Help]
I am unable to access the productVariant.quantityAvailable field, it is always returned null. Even when using the Shopify GraphQL Explorer app:

<img width="1750" alt="image" src="https://github.com/Shopify/storefront-api-feedback/assets/8808/369f70f3-3d10-4441-8fe1-2754af7a6169">

I know that the app has this scope because when I installed the graphQL explorer app I gave it all permissions - this is just a test store, so I checked off every single possible permission:

![image](https://github.com/Shopify/storefront-api-feedback/assets/8808/214d1504-1ca2-428d-80ea-9db3a466ca4c)

Am I doing something wrong or can support for this be added? I have done several searches about this and have found that other people have had similar issues:

https://community.shopify.com/c/extensions/is-there-any-way-to-add-new-access-scope-to-checkout-ui/td-p/2141674

**Comment by johncraigcole** [comment]
Hey @marcgug 👋 . The GraphiQL app doesn't support selection of `unauthenticated_read_product_inventory` on install. I see someone has responded to your ask over on the [ui-extension repo](https://github.com/Shopify/ui-extensions/issues/1594) that for ui-extensions this is something that will be fixed in the future.
